content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,233 @@
+
+![CMake and CTest workflow](/images/user-guide/testing/getting-started/cmake_ctest_workflow.png)
+
+## CTest
+
+[CTest](https://cmake.org/cmake/help/latest/manual/ctest.1.html) is similar to other test frameworks like GoogleTest, PyTest, or JUnit; though it uses a higher-level generic interface. CTest registers each test as a string of commandline arguments, and reports on whether invoking those arguments succeeded (returned zero) without hanging. CTest can also coordinate running tests in parallel processes. Unlike other test runners, CTest is programming language agnostic by interfacing with the OS shell. However this means a notable feature is absent from CTest: other test frameworks typically provide a lower-level interface which directly invokes functions inside a specific programming language. To provide this feature, CTest is used to invoke lower-level test runners.
+
+To simplify using function-level test reporting, O3DE provides wrapper code to register tests for frameworks such as GoogleTest and PyTest into CTest. These wrappers enable writing tests which run on any operating system. XML files are generated to track the low-level results not reported in CTest, as well as artifacts such as log output and crash dumps.
+
+In short, all tests in O3DE use CTest as a high level test coordinator. When executed, CTest calls other lower-level test tools and then reports on their success.
+
+![CTest calls other test runners](/images/user-guide/testing/getting-started/ctest_to_runners.png)
+
+### Starting CTest
+
+CTest expects its working directory to be a CMake build directory, so be sure to first navigate to this directory:
+
+```shell
+cd <local_path_to>/o3de/build/<build_folder>
+ctest -C <build_configuration>
+```
+
+You may not want to run the commands above without also filtering the set of tests to run. Without a filter the command will run every test, and likely result in multiple hours of test execution! If you ever want to stop CTest, send an interrupt signal by selecting the terminal and pressing `CTRL + C` . The most commonly used suites are Smoke and Main, which have requirements to execute fast. The following ctest command filters to only tests in these two suites:
+
+```shell
+cd <local_path_to>/o3de/build/<build_folder>
+ctest -C <build_configuration> -L "(SUITE_smoke|SUITE_main)"
+```
+
+The build configuration should be one you have already built locally. Running the Smoke and Main suites for the `profile` build configuration on a Linux machine should look like:
+
+```shell
+cd user/github/o3de/build/linux
+ctest -C profile -L "(SUITE_smoke|SUITE_main)"
+```
```

Comment:
Suggestion to break up and reformat slightly in a way that might resolve some of this confusion and reduce redundancy. It's not perfect but it feels like a better flow to me:

````suggestion
CTest can also take a set of tests to run with the `-L` argument. These test suites should be contained in parenthesis (`(..)`) and have their names separated with a `|` character. The following example demonstrates this syntax for running the Main and Smoke suites for a `profile` build:

```shell
cd user/github/o3de/build/linux
ctest -C profile -L "(SUITE_smoke|SUITE_main)"
```

The Smoke and Main suites are required to be quick. Unless performing a full regression, performance, or project-specific test, we recommend running these suites.

{{< caution >}}
Without a filter the command will run every test, and likely result in multiple hours of test execution! If you ever want to stop CTest, send an interrupt signal by selecting the terminal and pressing `CTRL+C`.
{{< /caution >}}
````

Reply:
Good suggestion. Moved the caution to be at the top of this block.